src/components/Button/Button.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import s from 'components/Button/Button.module.scss';
+
+export const Button = ({ title, city, getCity }) => {
+  const onButtonClick = () => {
+    if (city === title) {
+      return;
+    }
+
+    localStorage.setItem('city', JSON.stringify(title));
```

Comment:
move all functions with local storage stuff to separate utils file

Reply:
Fix